test/RDA.jl

```diff
@@ -1,62 +0,0 @@
-module TestRDA
```

Comment:
Yes, that's https://github.com/JuliaStats/DataFrames.jl/issues/955. I removed these files to get them out of the way, but we should move them to another package ASAP and deprecate them here.